api/src/certification/session/application/session-mass-import-controller.js

```diff
@@ -41,14 +42,15 @@ const validateSessions = async function (request, h, dependencies = { csvHelpers
 };
 
 const getTemplate = async function (request, h) {
-  const certificationCenterId = request.params.certificationCenterId;
+  const { id: certificationCenterId, hasBillingMode } = await usecases.getMassImportTemplate({
```

Comment:
```suggestion
  const { id: certificationCenterId, hasBillingMode } = await usecases.getMassImportInformation({
```
Je suis pas fan du nom que je propose, mais c'est pas un template 😛 
Je sais pas où est le renommage du usecase 🤔

Reply:
C'était pas choquant d'utiliser getCertificationCenter comme nom d'ailleurs

Reply:
Je suis d'accord, où alors il aurait fallu construire le template dans le usecase ; j'ai envie de garder la notion de template pour pas confondre avec l'import de session en masse (qui suit le téléchargement du template) du coup je mets `getMassImportTemplateInformation`

Reply:
Pour moi c'est un template, pourquoi non?